snowshu/adapters/base_sql_adapter.py

```diff
@@ -218,8 +218,7 @@ def _get_filtered_schemas(self, filters: Iterable[dict]) -> List[_DatabaseObject
         # get all schemas in all databases
         filtered_schemas = []
         for db_rel in filtered_databases:
-            schemas = self._get_all_schemas(
-                database=db_rel.quoted(db_rel.database))
+            schemas = self._get_all_schemas(database=db_rel.database)
```

Comment:
Do we need to preserve the `quoted` call here?

Reply:
The `quoted` call is applied on the database in the overridden method in `SnowflakeAdapter`

Reply:
Got it. This change might be related to the issues with the PostgresAdapter on the target side since they both inherit from the `BaseSQLAdapter` class.